spec/index.bs

```diff
@@ -4731,84 +4731,84 @@ Issue: Define the exact copy semantics, by reference to common algorithms shared
         Required if there are multiple [=images=] (i.e. the depth is more than one).
 </dl>
 
-### <dfn dictionary>GPUBufferCopyView</dfn> ### {#gpu-buffer-copy-view}
+### <dfn dictionary>GPUBufferCopyLocation</dfn> ### {#gpu-buffer-copy-location}
 
 <script type=idl>
-dictionary GPUBufferCopyView : GPUTextureDataLayout {
+dictionary GPUBufferCopyLocation : GPUTextureDataLayout {
     required GPUBuffer buffer;
 };
 </script>
 
-A {{GPUBufferCopyView}} contains the actual [=texture=] data placed in a [=buffer=] according to {{GPUTextureDataLayout}}.
+A {{GPUBufferCopyLocation}} contains the actual [=texture=] data placed in a [=buffer=] according to {{GPUTextureDataLayout}}.
 
 <div algorithm class=validusage>
-<dfn abstract-op>validating GPUBufferCopyView</dfn>
+<dfn abstract-op>validating GPUBufferCopyLocation</dfn>
 
   **Arguments:**
-    - {{GPUBufferCopyView}} |bufferCopyView|
+    - {{GPUBufferCopyLocation}} |bufferCopyLocation|
 
   **Returns:** {{boolean}}
 
   Return `true` if and only if all of the following conditions are satisfied:
-    - |bufferCopyView|.{{GPUBufferCopyView/buffer}} must be a [=valid=] {{GPUBuffer}}.
-    - |bufferCopyView|.{{GPUTextureDataLayout/bytesPerRow}} must be a multiple of 256.
+    - |bufferCopyLocation|.{{GPUBufferCopyLocation/buffer}} must be a [=valid=] {{GPUBuffer}}.
+    - |bufferCopyLocation|.{{GPUTextureDataLayout/bytesPerRow}} must be a multiple of 256.
 
 </div>
 
-### <dfn dictionary>GPUTextureCopyView</dfn> ### {#gpu-texture-copy-view}
+### <dfn dictionary>GPUTextureCopyLocation</dfn> ### {#gpu-texture-copy-location}
 
 <script type=idl>
-dictionary GPUTextureCopyView {
+dictionary GPUTextureCopyLocation {
     required GPUTexture texture;
     GPUIntegerCoordinate mipLevel = 0;
     GPUOrigin3D origin = {};
     GPUTextureAspect aspect = "all";
```

Comment:
This field philosophically doesn't make sense to have in a "location" object.

Reply:
Doesn't it? If you are describing the address ("location") of something, then the aspect will be one of the components of that address.